mkosi/sandbox.py

```diff
@@ -196,7 +188,7 @@ def sandbox_cmd(
 
     for d in ("bin", "sbin", "lib", "lib32", "lib64"):
         if (p := tools / d).is_symlink():
-            cmdline += ["--symlink", p.readlink(), Path("/") / p.relative_to(tools)]
+            cmdline += ["--symlink", p.readlink(), Path('/') / p.relative_to(tools)]
```

Comment:
Nit: The change in quotes is the only change in this line. Changing that back makes the diff nicer
```suggestion
            cmdline += ["--symlink", p.readlink(), Path('/') / p.relative_to(tools)]
```